bin/validators.js

```diff
@@ -48,6 +48,28 @@ module.exports = {
     return Boolean(firstOption && firstOption.allowIndentationTabs);
   },
 
+  "unicorn/template-indent"({ options }) {
+    if (options.length === 0) {
+      return false;
+    }
+
+    const firstOption = options[0];
+
+    if (!firstOption) {
+      return false;
+    }
+
+    const tags = firstOption.tags || [];
+    const comments = firstOption.comments || [];
+
+    return (
+      (tags.includes("html") ||
+        tags.includes("css") ||
+        tags.includes("gql") ||
+        comments.includes("HTML")) === false
+    );
```

Comment:
We still need to cover `styled`, right? And markdown. Dive into the Prettier code to see exactly what is supported.

Reply:
styled doesn't do anything for me, nor handlebars, nor yaml, nor json

I think it's just the files you linked above

Reply:
> styled doesn't do anything for me

These seem to be the patterns that Prettier looks for:

https://github.com/prettier/prettier/blob/83b9f62db4111f7d6c3a7ebd2fc01fca237d9072/src/language-js/embed/css.js#L115-L137

As far as I understand, the default config for the ESLint rule is more relaxed, while Prettier is more specific, but there’s still overlap?

Reply:
I’ve now learned that Prettier only cares about stuff like ``styled.h1`stuff` ``, while that does not seem to be configurable with unicorn except using selectors (the plugin only cares about `` styled`stuff` `` by default, which does not conflict). So we’re all good.